pkg/apis/controlplane/types.go

```diff
@@ -57,7 +57,7 @@ type GroupMemberPod struct {
 	// Pod maintains the reference to the Pod.
 	Pod *PodReference
 	// IP maintains the IPAddress of the Pod.
-	IP IPAddress
```

Comment:
This seems an incompatible API group change. @tnqn

Reply:
The controlplane.v1beta2 API is here https://github.com/vmware-tanzu/antrea/pull/1405

Reply:
Having some offline discussion with Quan, I will rebase the change on v1beta2. Since the PR to upgrade v1beta1 to v1beta2 is not merged yet, will rebase the code after that PR is ready.